src/core/src/package_managers/ZypperPackageManager.py

```diff
@@ -451,28 +467,203 @@ def get_package_size(self, output):
     # endregion
 
     # region auto OS updates
+    def __init_constants_for_yast2_online_update_configuration(self):
+        self.yast2_online_update_configuration_os_patch_configuration_settings_file_path = '/etc/sysconfig/automatic_online_update'
+        self.yast2_online_update_configuration_apply_updates_identifier_text = 'AOU_ENABLE_CRONJOB'
+        self.yast2_online_update_configuration_auto_update_config_pattern_match_text = '="(true|false)"'
+        self.yast2_online_update_configuration_installation_state_identifier_text = "installation_state"
```

Comment:
Variable naming suggests need to enacapsulate into an object

Reply:
Resolved